fe/lib/components/BrandSelect/__fixtures__/brands.ts

```diff
@@ -0,0 +1,341 @@
+          },
+        ],
+      },
+    },
+  ],
+};
+
+export const mockPaginatedBrandsSecondPage = {
+  pageInfo: {
+    hasNextPage: true,
+    hasPreviousPage: true,
+    startCursor: startCursorPaginatedBrandsSecondPage,
+    endCursor: endCursorPaginatedBrandsSecondPage,
+  },
+  edges: [
+    {
+      node: {
+        id: {
+          value:
+            'global:advertisementBranding:hirerBranding:4YsFiFRekSvuKjLzRUtmZH',
+        },
+        name: 'RyanAir - plane',
+        images: [
+          {
+            typeCode: 'OriginalLogo',
+            url: 'https://image-service-cdn.seek.com.au/6057509549dc838d6c3c518f0557c77fdbbdb1a7',
+          },
+          {
+            typeCode: 'CoverImage',
+            url: 'https://image-service-cdn.seek.com.au/d41de53cab6df901401b054a1185c786b7ac6526',
+          },
+        ],
+      },
+    },
+    {
+      node: {
+        id: {
+          value:
+            'global:advertisementBranding:hirerBranding:2P8WhewQF4dnZNYRKMDv1r',
+        },
+        name: 'RyanAir - blue cover image',
+        images: [
+          {
+            typeCode: 'OriginalLogo',
+            url: 'https://image-service-cdn.seek.com.au/2a4d048b3844bd5f771d0b22d55e3bab125c53a6',
+          },
+          {
+            typeCode: 'CoverImage',
+            url: 'https://image-service-cdn.seek.com.au/092f3e7154538e84fbc39b169f8e6e74d9c3655e',
+          },
+        ],
+      },
+    },
+    {
+      node: {
+        id: {
+          value:
+            'global:advertisementBranding:hirerBranding:VQ9aAzkhogAMSw3AF31stC',
+        },
+        name: 'SEEK EDC',
+        images: [
+          {
+            typeCode: 'OriginalLogo',
+            url: 'https://image-service-cdn.seek.com.au/f154d446456c99cc3100df6df37676935f8bc340',
+          },
+          {
+            typeCode: 'CoverImage',
+            url: 'https://image-service-cdn.seek.com.au/870108007f0cf52865e3213ff6bc43a83b7948a1',
+          },
+        ],
+      },
+    },
+    {
+      node: {
+        id: {
+          value:
+            'global:advertisementBranding:hirerBranding:D557J8rpyMKpP9DxrWg2s',
+        },
+        name: 'Vicinity Research Test',
+        images: [
+          {
+            typeCode: 'OriginalLogo',
+            url: 'https://image-service-cdn.seek.com.au/3681be7f69896fe765b83f8e68e9a9ffc10ec463',
+          },
+          {
+            typeCode: 'CoverImage',
+            url: 'https://image-service-cdn.seek.com.au/e9465179f03c690a90df96f200a57922a3acdb43',
+          },
+        ],
+      },
+    },
+  ],
+};
+
+export const mockPaginatedBrandsLastPage = {
+  pageInfo: {
+    hasNextPage: false,
+    hasPreviousPage: true,
+    startCursor: startCursorPaginatedBrandsLastPage,
+    endCursor: endCursorPaginatedBrandsLastPage,
+  },
+  edges: [
+    {
+      node: {
+        id: {
+          value:
+            'global:advertisementBranding:hirerBranding:qVe8DCsXJP9zNHsmcmEyL',
+        },
+        name: 'Spotlight - test',
```

Comment:
1. Can we upload some more neutral branding examples? They could just be some public domain patterned/gradient backgrounds and such. We shouldn't include the brands of real organizations or Ryanair here.

Reply:
We should feed that back in to public test as well

Reply:
Do you think it's safe to use SEEK's logo? The brands in public test use some of the generic SEEK branding:
<https://image-service-cdn.seek.com.au/27ecca452d06d960b596aa40380f0d06c56b6ba8>
<https://image-service-cdn.seek.com.au/7a2b5aa53d67c7c6d0eeae95bfa56d9b2f5ac228>

Reply:
@etaoins where can I get some more of those images as your example? or do I need to upload some of them as the sample brands then get the url from there?

Reply:
I think they all initially came from the Ryanair test hirer. You should be able use Web Inspector to get the out of Ryanair, or run `yarn printToken:prod` and run `advertisementBrandings` yourself. The issue is avoiding the third party & joke brands in Ryanair.

Reply:
yeah, I see what you meant, I also actually copied the responses from queries in ryanair to make them as mocked data, but looks like it includes some information which is not good for mocked data. 

What I meant is that if I want to use some other covers for my mocked data then where I can get them more. @72636c gave me the link to generate some more in [public test data](https://github.com/SEEK-Jobs/indie-public-test-data/blob/da9c02fc0b7dcb8952fa3df1358310f13e131065/mock-http-server/httpMocks/bx-resource-api/brands/mockBrands.js#L69), I'll look into it to generate some more to demonstrate pagination

Reply:
Ah yeah, I think if you create them on `talent.seek.com.au` they'll get uploaded to the CDN permanently.

I wonder if you could use some of these as cover images (they're free for commercial use):
<https://unsplash.com>

Reply:
Update mocked brands with Seek logo and some random free covers. I'll feed back some brands to public test data later